src/webgpu/api/operation/texture_view/write.spec.ts

```diff
@@ -351,6 +351,8 @@ TODO: Test rgb10a2uint when TexelRepresentation.numericRange is made per-compone
         t.skipIfTextureFormatNotUsableAsStorageTexture(format);
         break;
     }
+
+    t.selectDeviceForRenderableColorFormatOrSkipTestCase(format);
```

Comment:
Isn't this only if the method is `render-pass-resolve`, `render-pass-store`?

Reply:
Fixed. Also fix multisample issue